tests/panels/test_sql.py

```diff
@@ -122,6 +122,18 @@ def test_param_conversion(self):
             '["2017-12-22 16:07:01"]'
         ))
 
+    @unittest.skipUnless(connection.vendor not in ('sqlite', 'postgresql'), '')
+    def test_binary_param_force_text(self):
+        self.assertEqual(len(self.panel._queries), 0)
+
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT * FROM auth_user WHERE username = %s", [b'\xff'])
+
+        self.assertEqual(len(self.panel._queries), 1)
```

Comment:
Is it possible to assert the contents of the query? I think that would make the test more interesting.